torchsde/_core/sdeint.py

```diff
@@ -17,44 +17,41 @@
 
 import torch
 
-from . import adjoint_sde
 from . import base_sde
 from . import methods
-from .._brownian import BaseBrownian, TupleBrownian, BrownianInterval
+from . import misc
+from .._brownian import BaseBrownian, BrownianInterval
 from ..settings import SDE_TYPES, NOISE_TYPES, METHODS, LEVY_AREA_APPROXIMATIONS
 from ..types import TensorOrTensors, Scalar, Vector
 
 
-def sdeint(sde: [base_sde.BaseSDE],
+def sdeint(sde: base_sde.BaseSDE,
            y0: TensorOrTensors,
```

Comment:
This should just be a tensor now. I've not checked the whole codebase exhaustively, but do we need TensorOrTensors any more?

Reply:
Done.